hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -783,6 +783,16 @@ public ScmInfo getScmInfo() throws IOException {
     }
   }
 
+  @Override
+  public int renewDeletedBlockRetryCount(List<Long> txIDs) throws IOException {
+    getScm().checkAdminAccess(getRemoteUser());
+    AUDIT.logWriteSuccess(buildAuditMessageForSuccess(
+        SCMAction.RENEW_DELETED_BLOCK_RETRY_COUNT, null));
```

Comment:
Audit should log success only after the operation is done.  Exceptions should be logged as failure (and re-thrown).  Both should include the command's parameters.  See `listContainer` for an example.

https://github.com/apache/ozone/blob/58c3166094cc27f65da49f5edd5a12ff1103a96e/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java#L492-L549